Add `osu! Malaysia @ AniManGaki 2023` news post (#9928)
ppy/osu-wiki: news/2023/2023-08-22-osu-malaysia-animangaki.md

---
layout: post
title: "osu! Malaysia @ AniManGaki 2023"
date: 2023-08-22 08:00:00 +0000
---

The osu! Malaysia community proudly presents its first community-run space happening at [AniManGaki 2023](https://www.animangaki.com/)!

![](/wiki/shared/news/2023-08-22-osu-malaysia-animangaki/banner.jpg)

> The event will be happening at the Mines International Exhibition & Convention Centre Malaysia (*MIECC*) from 25th to 27th August 2023, providing an opportunity for osu! players all around the country to meet up and mingle!

Reviewer: https://miecc.mines.com.my/about/

.. although whether "mines" is capitalised seems inconsistent between sources <.<

```suggestion
The event will be happening at the MINES International Exhibition & Convention Centre (*MIECC*) in Malaysia from 25 to 27 August 2023, providing an opportunity for osu! players all around the country to meet up and mingle!
```

> ## LAN Tournament

Reviewer: ```suggestion
## LAN tournament
```

> The main highlight of the event includes the final bracket stages of the **osu! Malaysian LAN Tournament** for all four game modes, with each of the top 4 competing live at the following times (UTC+8):

Reviewer: ```suggestion
The main highlight of the event includes the final bracket stages of the **osu! Malaysian LAN Tournament** for all four game modes, with each of the top 4 competing live at the following times:
```

> - **Friday, 3:00PM:** osu!catch
> - **Saturday, 12:00PM:** osu!taiko
> - **Saturday, 3:00PM:** osu!mania
> - **Sunday, 2:00PM:** osu!

Reviewer: ```suggestion
| Game mode | Timestamp (UTC+08) |
| :-- | :-- |
| osu!catch | Friday, 25 August 2023 (15:00) |
| osu!taiko | Saturday, 26 August 2023 (12:00) |
| osu!mania | Saturday, 26 August 2023 (15:00) |
| osu! | Sunday, 27 August 2023 (14:00) |
```

Prizes for the winners of each mode includes exclusive booth merchandises and physical trophies!

> ## Booth Activities

Reviewer: ```suggestion
## Booth activities
```

Besides the exciting tournament matches, several other activities such as daily offline leaderboards, score challenges, contests, giveaways and more will be happening at the booth for everyone to enjoy, allowing for many new memories to be made together!

> For all the latest news on the booth happenings throughout the event, do visit the official [o!MY@AMG2023 Discord Server](https://discord.gg/ZKKWb8jK2B) or follow [@osumalaysia on Twitter](https://twitter.com/osumalaysia).

Reviewer: ```suggestion
For all the latest news on the booth happenings throughout the event, do visit the official [osu! Malaysia @ AMG2023 Discord server](https://discord.gg/ZKKWb8jK2B) or follow [@osumalaysia on Twitter](https://twitter.com/osumalaysia).
```
